app/src/main/java/com/example/myapplication/HomePage.java

```diff
@@ -78,13 +81,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        /*binding.button2.setOnClickListener(new View.OnClickListener() {
+
+        ViewModelProvider viewModelProvider = new ViewModelProvider(requireActivity());
+        UserModel userModel = viewModelProvider.get(UserModel.class);
+        MatchData matchData = new MatchData();
+        userModel.setMatchData(matchData);
+        binding.previousInfoButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 NavHostFragment.findNavController(HomePage.this)
                         .navigate(R.id.action_HomePage_to_pitFragment);
             }
-        });*/
+        });
+        binding.nextInfoButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UserModel.getMatchData().setBroke(true);
```

Comment:
This looks like a test place to update the 'broke' status; consider updating the actual place where the 'broke' status is stored (currently in SecondFragment.java)